_develop/connect/get-started.md

```diff
@@ -162,4 +157,24 @@ if (userSession.isSignInPending()) {
   const userData = await userSession.handlePendingSignIn();
   // your user is now logged in.
 }
-```
\ No newline at end of file
+```
+
+## Design Guidance
+
+Blockstack is valuable to users, but it can also be a barrier to those unfamiliar with Blockstack. The following guidelines serve to remedy that and help you onboard as many new users as you can. 
+
+### Delay Blockstack onboarding as long as possible
```

Comment:
should we educate them on how to refer/call the login? "Login using Secret Key" or "Login with Blockstack"?

Reply:
The recommendation is to just say "Get Started". I believe we had findings that "Login with X" doesn't work when you don't know what X is. Hence the introductory modal, etc

Reply:
The screenshot below includes this guidance, but I'll add it to the copy as well so that it's even clearer and text-based for SEO, etc.